.travis.yml

```diff
@@ -1,31 +1,32 @@
-language: python
-
-python:
-  - "3.5"
-
-env:
-  - PYTHON_TEST_VERSION=2.7
-  - PYTHON_TEST_VERSION=3.5
-  - PYTHON_TEST_VERSION=3.6
-
+dist: trusty
+matrix:
+  include:
+  - os: osx
+    env: PYTHON=3.6 NUMPY=1.12
```

Comment:
👍 - We should carry this NUMPY PYTHON build matrix to `earthio` as well.  `build_elm_env.sh` calls `build_earthio_env.sh`